### ISSUE TITLE
Access Dashboard

### ISSUE THREAD

**Issue by jack8387** [Platoniq/decidim-install#65]
Hi,

When i create a admin user, i can't acces to dashboard ?

How i can give access on dashboard to this user ?

Thanks

**Comment by microstudi** [comment]
When you say dahboard you mean the admin for the organization? aka `/admin/` ?
you should be able to access with the admin user you've created in `/system/`

**Comment by jack8387** [comment]
Hello,

Yes, 

For the first time i connect to /system/ and i create an admin
After, when i want to connect /admin/ the page users/sign_in it's send and if i try to connect with the admin create i can't, it's say invalid email/password. But if i connect /system/, no problem, it's ok

**Comment by jack8387** [comment]
The only way i find, it's to update manually the table and change the flag admin to t (By default it's F for False)

**Comment by microstudi** [comment]
that's ok, probably a mispeeling o caps lock activated during the admin creation

**Comment by zarpilla** [comment]
I don't think that were a mispeeling o caps lock problem, I've had the same problem right now.
